spyder/app/tour.py

```diff
@@ -144,68 +144,74 @@ def get_tour(index):
              'interact': False},
             ]
 
-    intro = [{'title': _("Welcome to the Introduction tour"),
+    intro = [{'title': _("Welcome to Spyder!"),
               'content': _("<b>Spyder</b> is a powerful Interactive "
                            "Development Environment (or IDE) for the Python "
                            "programming language.<br><br>"
-                           "Here we are going to guide you through its most "
+                           "Here, we are going to guide you through its most "
                            "important features.<br><br>"
                            "Please use the arrow keys or click on the buttons "
                            "below to move along the tour."),
               'image': 'tour-spyder-logo.png'},
 
              {'title': _("The Editor"),
-              'content': _("This is the pane where you write Python code before "
-                           "evaluating it. You can get automatic suggestions "
-                           "and completions while writing, by pressing the "
-                           "<b>Tab</b> key next to a given text.<br><br>"
-                           "The Editor comes "
-                           "with a line number area (highlighted here in red), "
-                           "where Spyder shows warnings and syntax errors. They "
-                           "can help you to detect potential problems before "
-                           "running the code.<br><br>"
+              'content': _("This is where you write Python code before "
+                           "evaluating it. You can get automatic "
+                           "completions while typing by pressing the "
+                           "<b>Tab</b> key, or automatically by enabling "
+                           "on-the-fly completions in Spyder's Preferences. "
+                           "<br><br>The Editor comes "

```

Comment:
This is the default now, i.e. you'll get automatic completions while typing. So, perhaps we should rephrase this?

Reply:
Sorry, I didn't remember it was on by default; despite finding it really useful in Rstudio, I've had it disabled due to the performance impacts and UX issues I and others have reported with it. I'll revise it accordingly.

Reply:
I removed the unnecessary text, and added brief mention of calltips and hover hints as other helpful introspection features (particularly for newer users) that are on by default.